src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalysisState.cs

```diff
@@ -148,7 +155,7 @@ private void OnCompilationEventsGenerated_NoLock(ImmutableArray<CompilationEvent
                             }
                             else
                             {
-                                newPartialSymbols = newPartialSymbols ?? ArrayBuilder<ISymbol>.GetInstance();
+                                newPartialSymbols ??= ArrayBuilder<ISymbol>.GetInstance();
```

Comment:
📝 This pull request has a bunch of extraneous code style changes which complicate the review. It likely would have completed sooner if they were omitted.